omero/sysadmins/customization.txt

```diff
@@ -48,24 +48,32 @@ Top links menu
 
 :property:`omero.web.ui.top_links` adds links to the top header::
 
-    $ bin/omero config append omero.web.ui.top_links '["Figure", "webfigure"]'
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "_blank"}]'
     $ bin/omero config set omero.web.ui.top_links '["GRE", "http://lifesci.dundee.ac.uk/gre"]'
 
 .. figure:: /images/topLink.png
 
+Open With option
+----------------
+
+:property:`omero.web.open_with` adds entry to the Open with right-click menu in tree::
+
+    $ bin/omero config append omero.web.open_with '["omero_figure", "new_figure",
+      {"supported_objects":["images"], "target": "_blank", "label": "OMERO.figure"}]'
+
 
 Group and Users in dropdown menu
 --------------------------------
 
 Customize the groups and users dropdown menu by changing the labels or hiding
 the entire list::
 
-    $ bin/omero config set omero.client.ui.menu.dropdown.leaders "Owners"
-    $ bin/omero config set omero.client.ui.menu.dropdown.colleagues.enabled true
-    $ bin/omero config set omero.client.ui.menu.dropdown.colleagues "Members"
-    $ bin/omero config set omero.client.ui.menu.dropdown.colleagues.enabled true
-    $ bin/omero config set omero.client.ui.menu.dropdown.all "All Members"
+    $ bin/omero config set omero.client.ui.menu.dropdown.leaders.label "Owners"
+    $ bin/omero config set omero.client.ui.menu.dropdown.leaders.enabled true
+    $ bin/omero config set omero.client.ui.menu.dropdown.colleagues.label "Members"
     $ bin/omero config set omero.client.ui.menu.dropdown.colleagues.enabled true
+    $ bin/omero config set omero.client.ui.menu.dropdown.everyone.label "All Members"
+    $ bin/omero config set omero.client.ui.menu.dropdown.everyone.enabled true
```

Comment:
Since the text says 'or hiding the entire list' do we want to have any of the ```enabled``` configs set to ```false```?
Do we also need an "update" page to tell users to upgrade, or will it be enough for them to realise something has changed and to find the new docs?